packages-web/maps/src/Maps.editorConfig.ts

```diff
@@ -0,0 +1,220 @@
+import { DynamicMarkersType, MapsPreviewProps, MarkersType } from "../typings";
+import { hideProperty, Problem, Properties } from "@widgets-resources/piw-utils";
+
+export function getProperties(
+    values: MapsPreviewProps,
+    defaultProperties: Properties,
+    target: "web" | "desktop"
+): Properties {
+    const defaultPropertiesForMarkers = defaultProperties?.[0].propertyGroups?.[0].properties?.[0].properties; // First config, first group, first property
+    const defaultPropertiesForDynamicMarkers = defaultProperties?.[0].propertyGroups?.[0].properties?.[1].properties; // First config, first group, second property
+    // console.log(JSON.stringify(defaultProperties));
+    // console.log(target); The epic is still waiting to be merged by PageEditor
+    if (!values.advanced) {
+        hideProperty<MarkersType>("markerStyle", defaultProperties);
+        hideProperty<DynamicMarkersType>("markerStyleDynamic", defaultProperties);
+        hideProperty<MarkersType>("customMarker", defaultProperties);
+        hideProperty<MapsPreviewProps>("mapProvider", defaultProperties);
+        hideProperty<MapsPreviewProps>("mapStyles", defaultProperties);
+        hideProperty<MapsPreviewProps>("geodecodeApiKey", defaultProperties);
+    }
+
+    values.markers.forEach((f, index) => {
+        const properties = defaultPropertiesForMarkers?.[index];
+        if (f.dataSourceType === "static") {
+            hideProperty<MarkersType>("propertyContext", properties);
```

Comment:
It could be really nice if we could write a wrapper function to hideProperty where we could pass array of "keys" to hide